pkg/testers/ginkgo/ginkgo.go

```diff
@@ -94,9 +101,109 @@ func (t *Tester) pretestSetup() error {
 	}
 	log.Printf("Using kubeconfig at %s", t.kubeconfigPath)
 
+	if err := t.AcquireTestPackage(); err != nil {
+		return fmt.Errorf("failed to get ginkgo test package from published releases: %s", err)
+	}
+
 	return nil
 }
 
+// TODO(michaelmdresser): change behavior if a local built e2e.test package
+// is available
+func (t *Tester) AcquireTestPackage() error {
+	releaseTar := fmt.Sprintf("kubernetes-test-%s-%s.tar.gz", runtime.GOOS, runtime.GOARCH)
+
+	downloadDir, err := ioutil.TempDir("", "kubetest2-ginkgo-download")
+	if err != nil {
+		return fmt.Errorf("failed to create temporary directory for download: %s", err)
+	}
+
+	defer func(dir string) {
+		if err := os.RemoveAll(dir); err != nil {
+			log.Printf("failed to remove temp dir %s used for release tar download: %s", dir, err)
+		}
+	}(downloadDir)
+
+	downloadPath := filepath.Join(downloadDir, releaseTar)
+
+	// first, get the name of the latest release (e.g. v1.20.0-alpha.0)
+
+	cmd := exec.Command("gsutil", "cat", "gs://kubernetes-release/release/latest.txt")
+	lines, err := exec.OutputLines(cmd)
+	if err != nil {
+		return fmt.Errorf("failed to get latest release name: %s", err)
+	}
+	if len(lines) == 0 {
+		return fmt.Errorf("getting latest release name had no output")
+	}
+
+	releaseName := lines[0]
+
+	// next, download the matching release tar
+
+	cmd = exec.Command("gsutil", "cp",
+		fmt.Sprintf(
+			"gs://kubernetes-release/release/%s/%s",
```

Comment:
nit: extract the gcs bucket to a flag (defaulting to `kubernetes-release`) for reusability.

Reply:
Do you mean let the user optionally specify a non-temp dir to download the tar to? It's currently deleted after extracting the test package and log messages include the version if the user wants to re-acquire it manually.

Reply:
no I meant the GCS bucket that's being used instead of a fixed one `gs://kubernetes-release` we should make that a flag.

Reply:
Done.